data/schemas/edges/coding_variants_phenotypes.VAMPAdapter.json

```diff
@@ -72,9 +72,19 @@
       ]
     },
     "method": {
-      "type": "string",
-      "description": "Method for the coding variant to phenotype edge",
-      "example": "VAMP-seq"
+      "enum": [
+        "VAMP-seq"
```

Comment:
schema needs to be updated according for biosample_term and biological_context. Also can you use edges base schema here. So we only need to add example or enum for _from, _to, name, inverse_name, source, source_url, label, method and class since they are already defined in edges base schema. One example schema is here: https://github.com/IGVF-DACC/igvf-catalog/blob/dev/data/schemas/edges/genomic_elements_biosamples.EncodeMPRA.json